.github/workflows/ci.yml

```
@@ -13,4 +13,21 @@ jobs:
     name: CI
     # Only run cron on the silverstripe account
     if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
-    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+    uses: maxime-rainville/gha-ci/.github/workflows/ci.yml@1
```

```suggestion
    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
```

Revert to this before merge.

---

This still needs to be reverted.  The corresponding gha-ci pr has been merged and released